README.md

```diff
@@ -20,10 +20,62 @@ Living outside of the core product, Kyma allows you to be completely language-ag
 
 Read the [documentation](docs/README.md) to learn about the product, its technology stack, and components.
 
+### Project structure
```

Comment:
hmmm, it will annoy me that this is more important than the installation instruction/links